.github/workflows/ci_e2e-mysql.yml

```
@@ -0,0 +1,194 @@
+name: End-to-End (MySQL)
+
+on:
+    workflow_dispatch: ~
+    workflow_call: ~
+
+jobs:
+    behat-no-js:
+        runs-on: ubuntu-latest
+        name: "Test Application without JS (PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}, Twig ${{ matrix.twig }}, DBAL ${{ matrix.dbal }}.x, API Platform ${{ matrix.api-platform }})"
```

```suggestion
        name: "Test Application without JS (PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}, Twig ${{ matrix.twig }}, DBAL ${{ matrix.dbal }}, API Platform ${{ matrix.api-platform }})"
```

the same change is needed in other builds. Check all names and leave there only needed dependencies.